client/src/components/InputsModal/InputsRadioGroup.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  FormControl,
+  FormControlLabel,
+  InputLabel,
+  ListItem,
+  Radio,
+  RadioGroup,
+} from '@mui/material';
+import LockIcon from '@mui/icons-material/Lock';
+import { TestInput } from 'models/testSuiteModels';
+import React, { FC, Fragment } from 'react';
+import useStyles from './styles';
+
+export interface InputRadioGroupProps {
+  requirement: TestInput;
+  index: number;
+  inputsMap: Map<string, string>;
+  setInputsMap: (map: Map<string, string>) => void;
+}
+
+const InputRadioGroup: FC<InputRadioGroupProps> = ({
+  requirement,
+  index,
+  inputsMap,
+  setInputsMap,
+}) => {
+  const styles = useStyles();
+  const [value, setValue] = React.useState(
+    inputsMap.get(requirement.name) || requirement.default || null
```

Comment:
It looks like `InputOption.value` could be a `number` or `boolean` according to the type definition.  So if the test writer sets a value to `0` or `false` on a radio option, this will fall back to the default or null if that option was selected?

I'm not sure how this actually comes through the API (maybe all values are cast to a string), but if they are all cast to a string then we probably should update the `InputOption` type definition. Or, if it really does allow those types, be more explcit in this logic and don't rely on falsey values.

(or please let me know if I'm inventing a problem that doesn't exist, I haven't actually tried this!)

Reply:
So this is actually handled "properly" by the frontend (e.g. if `InputOption.value` is a `number` or `boolean` in the backend, then the correct value is displayed) but it is handed back to the backend as a `string`. I think it makes sense to update the type so that all values are `string` to avoid unnecessary complications, unless you can think of a reason not to?